### DISCUSSION TITLE
*终于能绳网了*

### DISCUSSION THREAD

**Discussion by Miao-newuser** [share121/inter-knot#2763] [其他]
看看大伙都在忙些什么呢（盯<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by lixiao189** [comment]
忙着做委托呢，总有一天我会超越传奇绳匠法厄同！
